tools/testing/test_selections.py

```diff
@@ -48,14 +49,25 @@ def __init__(self) -> None:
         self.serial: List[ShardedTest] = []
         self.parallel: List[ShardedTest] = []
 
-    def get_total_time(self) -> float:
+    def _get_time_helper(self, get_test_time: Callable[[ShardedTest], float]) -> float:
```

Comment:
Fine for now, but this code organization seems a little unnecessarily complicated. This. helper can just take an arg and use `test.get_time` or the `get_assumed_time._get_time` logic in-line based on that arg. And what is the different btwn `test.get_time()` and `test.time`?

Reply:
`test.time` can be none, `test.get_time()` will always return a float.  Basically just a connivence thing because I don't want to type `time.time or 0` everywhere